### PR TITLE
test/resetNiaidPortal: set NIAID portal to master branch

### PR DESCRIPTION
**Pull request by ocshawn**
### Environments
NIAID

### Description of changes
set NIAID portal to master branch

### CODE REVIEW

File: qa-niaid.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.05",
     "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.05",
     "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.05",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_studyViewerIdpFlag",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:master",
```
Comment (paulineribeyre) [inline qa-niaid.planx-pla.net/manifest.json:21]:
```suggestion
    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.05",
```

Comment (ocshawn) [reply qa-niaid.planx-pla.net/manifest.json:21]: would rather set it to master so we can see the latest changes, or does that release need to be tested?

Comment (paulineribeyre) [reply qa-niaid.planx-pla.net/manifest.json:21]: oh i see, then let's create a new data-portal release and deploy it?

Comment (ocshawn) [reply qa-niaid.planx-pla.net/manifest.json:21]: ok ill do that after i merge my latest changes

**Comment by ocshawn** [comment]
not needed will go with the next monthly release

**Pull request closed without merging**